features/steps/project/source/browse_files.rb

```diff
@@ -106,8 +122,35 @@ class Spinach::Features::ProjectSourceBrowseFiles < Spinach::FeatureSteps
   end
 
   step 'I can see new file page' do
-    expect(page).to have_content "New file"
-    expect(page).to have_content "Commit message"
+    page.should have_content "new file"
+    page.should have_content "Commit message"
+  end
+  
+  step 'I can see "upload existing one"' do
+    page.should have_content "upload existing one"
+  end
+
+  step 'I click on "upload existing one"' do
+      click_link 'upload existing one'
+  end
+  
+  step 'I click on "Upload file"' do
+    click_button 'Upload file'
+  end
+  
+  step 'I upload "user.feature"' do
+    attach_file(:file_upload, File.join('features', 'user.feature'))
+  end
+  
+  step 'I check name of the uploaded file' do
+    ".gitignore" != "user.feature"
+    "LICENSE" != "user.feature"
+    "VERSION" != "user.feature"
```

Comment:
@liyakun when writing a test, first try to write a failing test. If the test can't fail, why would you write it? 

Maybe you can make "user.feature" or ".gitignore" a variable, rather than a string? Otherwise we're not testing functionality, integration or any code. We are just adding a test that always passes. There is no value in that.

So to repeat, try writing a test that fails something and then write the code that has to pass it. In this case, you could comment out or remove part of your code so that this test will fail. Then you write the code to make it pass and refactor it.

Here is a nice website on it: http://www.jamesshore.com/Blog/Red-Green-Refactor.html